modules/http/src/test/java/org/mule/module/http/functional/requester/HttpRequestMultipleValueHeadersTestCase.java

```diff
@@ -36,11 +39,13 @@
     public SystemProperty host = new SystemProperty("host", "localhost");
     @Rule
     public SystemProperty encoding = new SystemProperty("encoding" , CHUNKED);
+    @Rule
+    public DynamicPort port = new DynamicPort("port");
 
     @Override
     protected String getConfigFile()
     {
-        return "http-request-headers-config.xml";
```

Comment:
Yes, this test was actually reusing the config file of the general org.mule.module.http.functional.requester.HttpRequestHeadersTestCase so a created a specific one for it.